app/src/main/java/com/sopt/now/compose/data/remote/di/NetworkModule.kt

```diff
@@ -31,22 +37,41 @@ object NetworkModule {
 
     private val okHttpClient =
         OkHttpClient.Builder()
-            .addInterceptor(provideLogOkHttpClient())
+            .addInterceptor(getLogOkHttpClient())
             .build()
 
-    fun provideRetrofit(url: String): Retrofit {
+    @Singleton
+    @Provides
+    fun provideRetrofit(): Retrofit.Builder {
         return Retrofit.Builder()
-            .baseUrl(url)
             .client(okHttpClient)
             .addConverterFactory(json.asConverterFactory(CONTENT_TYPE.toMediaType()))
-            .build()
     }
 
-    inline fun <reified T> create(url: String): T = provideRetrofit(url).create(T::class.java)
-}
+    @Singleton
+    @Provides
+    fun provideAuthService(
+        retrofitBuilder: Retrofit.Builder,
+        @AuthBaseUrl url: String,
+    ): AuthService {
+        return retrofitBuilder.baseUrl(url).build().create(AuthService::class.java)
```

Comment:
```suggestion
    @Provides
    fun provideAuthService(
        retrofitBuilder: Retrofit.Builder,
    ): AuthService {
    val url = BuildConfig.AUTH_BASE_URL
        return retrofitBuilder.baseUrl(url).build().create()
```

url 을 Qualifier 를 활용하여 주입하신 이유가 있을까요?

Reply:
아직 Hilt로 Url 두개받는게 어색한 것 같습니다...! 😅